### PR TITLE
fix(DB): Add additional Incendia Agave spawns

### PR DESCRIPTION
**Pull request by vtihomirov**
## Changes Proposed:
-  Added 4 more Incendia Agave spawns according to the sources.

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/5152
- Closes https://github.com/chromiecraft/chromiecraft/issues/344

## SOURCE:

https://www.youtube.com/watch?v=bE1RIbdOaYE
https://www.youtube.com/watch?v=D62TUG1-mq8

## Tests Performed:

- Tested ingame if the objects are re-spawning on 180 sec.
- Tested if the correct item is received.
- Tested if the spawn positions aren't clipping with the ground.
- Tested on docker/linux.

## How to Test the Changes:

1. Go to each of these locations in thousand needles.
  - `.go xyz -4947.665527 -2019.104126 -52.501392`
  - `.go xyz -5031.855469 -2040.091675 -51.542252`
  - `.go xyz -5027.564941 -2005.562378 -53.395096`
  - `.go xyz -4993.604004 -1971.745728 -53.395203`
2. Loot the node while having quest(5062), you should get the quest item.
2. Loot the node while not having quest(5062), you shouldn't get the quest item.
3. Check if the respawn time is 180 seconds.
4. Check if the node is possitioned well and if its not in some way cliping with the ground.

## Known Issues and TODO List:

- [ ]
- [ ] 

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1619259290727738768.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1619259290727738768');
+
+SET @INCENDIA_AGAVE_ENTRY := 175928;
+INSERT INTO `gameobject` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`, `ScriptName`, `VerifiedBuild`) VALUES
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1619259290727738768.sql:4]: thanks for your contribution! please always use `DELETE` before `INSERT`

**Review by FrancescoBorzi — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1619259290727738768.sql (hunk 1 above)
Comment (FrancescoBorzi) [reply data/sql/updates/pending_db_world/rev_1619259290727738768.sql:4]: @vtihomirov friendly reminder :)

Comment (vtihomirov) [reply data/sql/updates/pending_db_world/rev_1619259290727738768.sql:4]: @FrancescoBorzi sorry for the delayed reply.
Two days ago I experienced a hardware failure on my dev machine and I won't be able to resolve this until I get my system up and running. Can't really give you ETA on this as I'm waiting on logistics to bring the new parts.

Cheers

Comment (FrancescoBorzi) [reply data/sql/updates/pending_db_world/rev_1619259290727738768.sql:4]: thanks for letting us know @vtihomirov and good luck with it!

File: data/sql/updates/pending_db_world/rev_1619259290727738768.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1619259290727738768');
+
+SET @INCENDIA_AGAVE_ENTRY := 175928;
+
+DELETE FROM `gameobject` WHERE (`id` = @INCENDIA_AGAVE_ENTRY) AND (`guid` IN (2135276, 2135277, 2135278, 2135279));
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1619259290727738768.sql:5]: These are very high GUIDs...

Use 

6024,
6025,
6026,
6027,

Instead. This way we can fill gaps in the DB.

You can use tools like https://github.com/Kitzunu/Unused-Guid-Search/tree/master to find open entries

**Review by Kitzunu — CHANGES_REQUESTED** [review]

**Review by Kitzunu — APPROVED** [review]

**Comment by Kitzunu** [comment]
Thanks =)

**Pull request merged**